components.yaml

```diff
@@ -29,7 +29,8 @@ GMAO_Shared:
 MAPL:
   local: ./src/Shared/@MAPL
   remote: ../MAPL.git
-  tag: v2.6.4
+  tag: v2.6.7
+  branch: main
```

Comment:
@weiyuan-jiang I think modern mepo will not allow this. You can have a tag or a branch but not both!